app/src/main/res/values/strings.xml

```diff
@@ -213,6 +213,7 @@
     <string name="delete_search_history_alert">Delete entire search history?</string>
     <string name="search_history_deleted">Search history deleted</string>
     <string name="help">Help</string>
```

Comment:
You should remove the "Help" string completely (in all language files), unless it's used in some other place.

Reply:
Done

Reply:
It has only one instance, so I removed it from all language files.